test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -211,6 +213,116 @@ public void CreateDisplayMetadata_DisplayAttribute_DescriptionFromResources()
             Assert.Equal("description from resources", context.DisplayMetadata.Description());
         }
 
+        [Fact]
+        public void CreateDisplayMetadata_DisplayAttribute_NullLocalizer()
```

Comment:
How does this test differ from the other ones above where you have a null localizer?